packages/clerk-js/src/ui/common/authForms/Header.tsx

```diff
@@ -36,12 +35,21 @@ export function Header({
           handleClick={handleBack}
         />
       )}
-      {error && <Error style={errorStyle}>{error}</Error>}
+
+      {alerts.length > 0 && (
```

Comment:
❓ Do we show multiple alerts at the same time or is it always one? If no, let's just stick to a single Alert for now to make the API easier.

Reply:
🔧  Instead of passing an array of alerts, we can simply pass a component that renders multiple alerts if needed. I'd change HeaderProps to accept a `alert?: React.ReactNode;` prop instead. That'd simplify the common use cases like:  https://github.com/clerkinc/javascript/pull/186/files#diff-f552e096d1e5ff1ceb1055fa13af0712224e06969c8e494cac5aad867ecefde3R102

In case multiple alerts are necessary (
https://github.com/clerkinc/javascript/pull/186/files#diff-a365df1e8e318373fc6e01f7e89b201851d990bacd9724017b5c98bdd892b6f5R117), I'd wrap them in a fragment, eg: 
```
        alerts={<>
            {error && <Alert type='error'>{error}</Alert>}
            <Alert type='info'>Please fill in the following information to complete your sign-up</Alert>
        </>}
```

Reply:
@nikosdouvlis Will adapt as per your recommendation.

Reply:
Fixed